src/rsz/src/Resizer.cc

```diff
@@ -2391,10 +2391,9 @@ string Resizer::makeUniqueInstName(const char* base_name, bool underscore)
 {
   string inst_name;
   do {
-    stringPrint(inst_name,
-                underscore ? "%s_%d" : "%s%d",
-                base_name,
-                unique_inst_index_++);
+    // sta::stringPrint can lead to string overflow and crash
+    inst_name = fmt::format(
+        underscore ? "{}_{}" : "{}{}", base_name, unique_inst_index_++);
```

Comment:
Doing dynamic fmt patterns is discouraged/doesn't work on C++20.

Can you change this to an if where you either create one string or the other?

Reply:
@precisionmoon you can see that jenkins is failing in the C++20 build. Please follow what @QuantamHD suggested.

Reply:
Fixed

Reply:
Note the latent bug lurking in that code. The scope of the name (instance) should be passed in and not assumed to be the top instance within the makeUnique**** code. At the very least use a parameter defaulted to nullptr which implies top instance when unused.

Reply:
> Note the latent bug lurking in that code. The scope of the name (instance) should be passed in and not assumed to be the top instance within the makeUnique**** code. At the very least use a parameter defaulted to nullptr which implies top instance when unused.

@andyfox-rushc,. thanks for the heads up.   I'll modify these two routines plus others in another branch.

Reply:
Why do we even need the underscore option?  Its use seems haphazard.